src/bin/zmtrainsa

```diff
@@ -117,6 +117,13 @@ fi
 source `dirname $0`/zmshutil || exit 1
 zmsetvars
 
+export TMPDIR=${zmtrainsa_tmp_directory:-${zimbra_tmp_directory}/zmtrainsa}
```

Comment:
Because $zimbra_tmp_directory is world writable (mode 1777), it should be considered unsafe to hardcode the subdirectory "zmtrainsa".  Let's just default to:
`export TMPDIR=${zmtrainsa_tmp_directory:-${zimbra_tmp_directory}}`